src/main/java/com/hubspot/jinjava/tree/parse/DefaultTokenScannerSymbols.java

```diff
@@ -0,0 +1,58 @@
+package com.hubspot.jinjava.tree.parse;
+
+public class DefaultTokenScannerSymbols extends TokenScannerSymbols {
+  char TOKEN_PREFIX_CHAR = '{';
+  char TOKEN_POSTFIX_CHAR = '}';
+  char TOKEN_FIXED_CHAR = 0;
+  char TOKEN_NOTE_CHAR = '#';
+  char TOKEN_TAG_CHAR = '%';
+  char TOKEN_EXPR_START_CHAR = '{';
+  char TOKEN_EXPR_END_CHAR = '}';
+  char TOKEN_NEWLINE_CHAR = '\n';
+  char TOKEN_TRIM_CHAR = '-';
+
+  @Override
+  public char TOKEN_PREFIX_CHAR() {
```

Comment:
should be `getTokenPrefix()`. Same for the others below.

Reply:
changes made!